utils/dwdparse/dwdparse.go

```diff
@@ -80,7 +82,9 @@ func BuildArgsMap(dwd string) (map[string]string, error) {
 		}
 
 		if len(keyValue) == 1 {
-			argsMap[keyValue[0]] = "true"
+			// We won't know how to interpret an empty value--whether it's allowed,
+			// or what its type should be--until ValidateArgs().
+			argsMap[keyValue[0]] = emptyValue
```

Comment:
Does this not work if you set `emptyValue=""` ?

Reply:
Yep, that works, too. I'd rather not do that; if something goes bad at least we'll get my empty_value string dumped out somewhere as a clue.